packages/sync-rules/src/TsSchemaGenerator.ts

```diff
@@ -3,26 +3,53 @@ import { SchemaGenerator } from './SchemaGenerator.js';
 import { SqlSyncRules } from './SqlSyncRules.js';
 import { SourceSchema } from './types.js';
 
+export interface TsSchemaGeneratorOptions {
+  language?: 'ts' | 'js';
+  imports?: 'web' | 'react-native' | 'auto';
```

Comment:
Is there a reason you're not using an enum?

Reply:
Laziness mostly